### PR TITLE
feat: add paysWithFeeOrigin support for MultiLocations

### PR DESCRIPTION
**Pull request by TarikGul**
## Summary

- This updates polkadot-js to `10.11.1`.
- Allow for both multilocation and integer input for `paysWithFeeOrigin`.
- Add transferAllowDeath support for balances::transfer
- Update metadata for testing for system parachains

### CODE REVIEW

File: src/config/disabledOpts.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const disabledOpts: DisabledOptions = {
 	},
 	paysWithFeeOrigin: {
 		disabled: true,
```
Comment (bee344) [inline src/config/disabledOpts.ts:29]: Shouldn't this just be `disabled: false`

Comment (TarikGul) [reply src/config/disabledOpts.ts:29]: Yup nice catch.

Comment (TarikGul) [reply src/config/disabledOpts.ts:29]: fixed

**Review by bee344 — APPROVED** [review]
Nice job

**Comment by TarikGul** [comment]
rel: https://github.com/paritytech/asset-transfer-api/issues/228

**Pull request merged**